src/rentACar/Application/Features/UserOperationClaims/Rules/UserOperationClaimBusinessRules.cs

```diff
@@ -19,4 +19,16 @@ public async Task UserOperationClaimIdShouldExistWhenSelected(int id)
         UserOperationClaim? result = await _userOperationClaimRepository.GetAsync(b => b.Id == id);
         if (result == null) throw new BusinessException(UserOperationClaimMessages.UserOperationClaimNotExists);
     }
+
+    public async Task ThisOperationClaimCannotBeDuplicatedForThisUserWhenCreated(int userId, int operationCliamId)
+    {
+        UserOperationClaim? result = await _userOperationClaimRepository.GetAsync(uoc => uoc.UserId == userId && uoc.OperationClaimId == operationCliamId);
+        if (result != null) throw new BusinessException(UserOperationClaimMessages.ThisUserAlreadyHasThisOperationClaim);
+    }
+
+    public async Task ThisOperationClaimCannotBeDuplicatedForThisUserWhenUpdated(int id, int userId, int operationCliamId)
```

Comment:
Fix typo

Reply:
Also, this method is not needed, the same situation is checked in insert and update.